README.md

```
@@ -32,7 +32,7 @@
 ## Table of Contents
 
 - [📖 Documentation](#-documentation)
-- [🔧 Installation](#-installation)
+- [🔧 Installation (>= 1.20)](#-installation-120)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)